packages/lesswrong/server/deleteUserContent.ts

```diff
@@ -31,6 +31,10 @@ export const deleteUserContent = async (
 
   const userComments = await Comments.find({ userId, deleted: false }).fetch();
 
+  if (userComments.length > 1000) {
+    throw new Error(`About to delete ${userComments.length} comments, please double-check that you want to do this and comment this line out if so!`);
```

Comment:
Hmm. This seems like a fairly janky way of doing this, but I guess maybe it's fine given how often this is gonna come up.

Reply:
Yep!